ocpp/v201/datatypes.py

```diff
@@ -555,7 +555,7 @@ class UnitOfMeasureType:
     UnitOfMeasureType is used by: SampledValueType
     """
 
-    unit: Optional[str] = None
+    unit: Optional[enums.StandardizedUnitsOfMeasureType] = None
```

Comment:
<img width="830" alt="image" src="https://github.com/user-attachments/assets/c72e8e7c-f4fe-42c0-9e7f-740609d4b342">

Will this change adhere to the specs from OCPP docs? How can I define `custom` unit

Reply:
I find this part of the spec confusing. They emphasize *SHALL* use from the list, which implies those are the only valid ones. Then go on to say that one can create a custom unit?

Either way, I've updated the type hint to:
`unit: Optional[Union[enums.StandardizedUnitsOfMeasureEnumType, str]] = None